build/SignToolData.json

```diff
@@ -4,38 +4,57 @@
       "certificate": "MicrosoftSHA2",
       "strongName": "MsSharedLib72",
       "values": [
-        "Analyzer.Utilities/netstandard1.3/Analyzer.Utilities.dll",
-        "Microsoft.CodeAnalysis.Analyzers/netstandard1.3/Microsoft.CodeAnalysis.Analyzers.dll",
-        "Microsoft.CodeAnalysis.CSharp.Analyzers/netstandard1.3/Microsoft.CodeAnalysis.CSharp.Analyzers.dll",
-        "Microsoft.CodeAnalysis.VisualBasic.Analyzers/netstandard1.3/Microsoft.CodeAnalysis.VisualBasic.Analyzers.dll",
-        "Microsoft.CodeQuality.Analyzers/netstandard1.3/Microsoft.CodeQuality.Analyzers.dll",
-        "Microsoft.CodeQuality.CSharp.Analyzers/netstandard1.3/Microsoft.CodeQuality.CSharp.Analyzers.dll",
-        "Microsoft.CodeQuality.VisualBasic.Analyzers/netstandard1.3/Microsoft.CodeQuality.VisualBasic.Analyzers.dll",
-        "Microsoft.NetCore.Analyzers/netstandard1.3/Microsoft.NetCore.Analyzers.dll",
-        "Microsoft.NetCore.CSharp.Analyzers/netstandard1.3/Microsoft.NetCore.CSharp.Analyzers.dll",
-        "Microsoft.NetCore.VisualBasic.Analyzers/netstandard1.3/Microsoft.NetCore.VisualBasic.Analyzers.dll",
-        "Microsoft.NetFramework.Analyzers/netstandard1.3/Microsoft.NetFramework.Analyzers.dll",
-        "Microsoft.NetFramework.CSharp.Analyzers/netstandard1.3/Microsoft.NetFramework.CSharp.Analyzers.dll",
-        "Microsoft.NetFramework.VisualBasic.Analyzers/netstandard1.3/Microsoft.NetFramework.VisualBasic.Analyzers.dll",
-        "Roslyn.Diagnostics.Analyzers/netstandard1.3/Roslyn.Diagnostics.Analyzers.dll",
-        "Roslyn.Diagnostics.CSharp.Analyzers/netstandard1.3/Roslyn.Diagnostics.CSharp.Analyzers.dll",
-        "Roslyn.Diagnostics.VisualBasic.Analyzers/netstandard1.3/Roslyn.Diagnostics.VisualBasic.Analyzers.dll",
-        "Text.Analyzers/netstandard1.3/Text.Analyzers.dll",
-        "Text.CSharp.Analyzers/netstandard1.3/Text.CSharp.Analyzers.dll",
-        "Text.VisualBasic.Analyzers/netstandard1.3/Text.VisualBasic.Analyzers.dll"
+        "bin/Analyzer.Utilities/netstandard1.3/Analyzer.Utilities.dll",
+        "bin/Analyzer.Utilities/netstandard1.3/*/Analyzer.Utilities.resources.dll",
+        "bin/Microsoft.CodeAnalysis.Analyzers/netstandard1.3/Microsoft.CodeAnalysis.Analyzers.dll",
+        "bin/Microsoft.CodeAnalysis.Analyzers/netstandard1.3/*/Microsoft.CodeAnalysis.Analyzers.resources.dll",
+        "bin/Microsoft.CodeAnalysis.CSharp.Analyzers/netstandard1.3/Microsoft.CodeAnalysis.CSharp.Analyzers.dll",
+        //"bin/Microsoft.CodeAnalysis.CSharp.Analyzers/netstandard1.3/*/Microsoft.CodeAnalysis.CSharp.Analyzers.resources.dll",
```

Comment:
I think all the resources are in the shared Core analyzer assembly. C# and VB assemblies don't have resources.